src/App.tsx

```diff
@@ -1,26 +1,43 @@
-import { useSelector } from 'react-redux';
-
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import './App.scss';
-import Start from './components/Start';
-import { Finish } from './components/Finish';
+import './styles/general.scss';
+import { TodoList } from './components/TodoList';
+import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api';
+import { getTodosSelector, getUserIdSelector } from './store';
+
+const App: React.FC = () => {
+  const selectedUserId = useSelector(getUserIdSelector);
+  const todos = useSelector(getTodosSelector);
+  const dispatch = useDispatch();
 
-import { isLoading, getMessage } from './store';
+  useEffect(() => {
+    const todosFromServer = async () => {
+      dispatch({ type: 'TODOS_LOADING', todos: await getTodos() });
+    };
 
-const App = () => {
-  const loading = useSelector(isLoading);
-  const message = useSelector(getMessage) || 'Ready!';
+    todosFromServer();
+  }, []);
 
   return (
     <div className="App">
-      <h1>Redux list of todos</h1>
-      <h2>{loading ? 'Loading...' : message}</h2>
+      <div className="App__sidebar">
+        <TodoList
+          todos={todos}
+          selctedUser={selectedUserId}
```

Comment:
Don't need to use props here. You can use useSelector on TodoList component for get todos and selectedUserId.